evap/static/scss/components/_infobox.scss

```diff
@@ -0,0 +1,13 @@
+.infobox.callout {
```

Comment:
This is completely duplicated from [evap/static/scss/components/_callouts.scss](https://github.com/e-valuation/EvaP/pull/1833/commits/3c5360a2f4aaca044596e2169b53083baf95a639#diff-97a87ac7f4b8ff763d6303984990fb98dbb69f65dbf3b54f8f73ce87a9133d81). Shouldn't be necessary, and even if including the rules in both files was necessary, we'd want to have them only in one .scss file.